### ISSUE TITLE
Profil: E-Mail-Adresse lässt sich nicht ändern, obwohl Feld editierbar

### ISSUE THREAD

**Issue by geraldk1** [valuehack/scholarium.at#120]
Mögliche Lösungen:

- Feld ausgrauen, d.h. nicht editierbar machen
- Änderung ermöglichen

**Comment by wmles** [comment]
ja, das Feld sollte nicht editierbar machen und stattdessen ein Link auf /nutzer/<username>/email sein
